workbench-core/accounts/src/services/projectService.test.ts

```diff
@@ -1494,6 +1496,284 @@ describe('ProjectService', () => {
     });
   });
 
+  describe('updateProject', () => {
+    let projectName: string = project1.name;
```

Comment:
jest recommends keeping all assignments in the `beforeEach` to prevent any race conditions or unexpected sequencing when prepping the test data. Can you do that for these?

Reply:
Yup. I just wanted to give them defaults but I can do that in the `beforeEach` I suppose.